### DISCUSSION TITLE
Contributing Tailwind v4 config code

### DISCUSSION THREAD

**Discussion by zensitively** [shadcn-ui/ui#6045] [Ideas]
Hello,

I wanted to ask if it is desired / welcome to contribute the Tailwind v4 config code as an alternative to the old config format. In the long run, it could be possible to select with the copy button which version of config to select in tailwind.

At the moment, I have to opt for the manual installation when using tailwind 4 (for example with the accordion component) as it cannot find the tailwind config file it is looking for.

And the config itself has to be different, like so:

```
@keyframes accordion-down {
  from { height: 0 }
  to { height: var(--radix-accordion-content-height) }
}

@keyframes accordion-up {
  from { height: var(--radix-accordion-content-height) }
  to { height: 0 }
}

@animation accordion-down {
  animation: accordion-down 0.2s ease-out;
}

@animation accordion-up {
  animation: accordion-up 0.2s ease-out;
}
```

**Comment by georgeeburt** [comment]
Hi, please check out the contribution guide:

https://github.com/shadcn-ui/ui/blob/main/CONTRIBUTING.md
